### ISSUE TITLE
Render package price on a product page

### ISSUE THREAD

**Issue by duker33** [fidals/shopelectro#827]
If product has a related tag "В упаковке" with value, render a price per pack, not price per unit.
For example for product from a screen you should change price `524 руб / шт` to `2096 руб / упак`, because this product contains four units in a package.

![image](https://user-images.githubusercontent.com/3526796/56790192-95955980-680c-11e9-9205-10ca709eedb5.png)

**Comment by artemiy312** [comment]
Tags from the group
```
1 в блистере
2 в блистере
3+1 в блистере
3 в блистере
4+1 в блистере
4 в блистере
4+2 в блистере
5 в блистере
5+1 в блистере
6 в блистере
6+2 в блистере
8+2 в блистере
8 в блистере
10 в блистере
10+2 в блистере
12 в блистере
16 в блистере
18 в блистере
20 в блистере
24 в блистере
28 в блистере
30 в блистере
1 в коробке
2 в коробке
4 в коробке
5 в коробке
6 в коробке
10 в коробке
12 в коробке
20 в коробке
24 в коробке
в пакете
4 в пластмассовом боксе
1 в стяжке
2 в стяжке
4 в стяжке
8 в стяжке
10 в стяжке
1 шт
2 шт
3 шт
4 шт
6 шт
10 шт
12 шт
```

**Comment by artemiy312** [comment]
We have two options:
1) We would create new model `ProductPack` or field `in_pack` for `Product`.
But I have a concern about pull process from 1C database, It will probably make it more complex

2) We would  make a specific query to database to rearrange the price.
The data are heterogeneous, but we can get the number as we gets it [here](https://github.com/fidals/refarm-site/blob/master/catalog/models.py#L292). 
We will face an issue with such `8+2 в блистере` tags, but we can just rename or merge them with existence. (e.g. `8+2 в блистере -> 10 в блистере`)

I'd prefer the first option, but i don't sure

@duker33 What do you think?

**Comment by duker33** [comment]
@artemiy312 , the first looks more good for me too.
The task is complex by it's nature. We are choosing where to put it: to the update_catalog or to the product rendering.
I preper to hide complexity in `update_catalog` as smth more "static"

**Comment by 0pdd** [comment]
@duker33 2 puzzles [#858](https://github.com/fidals/shopelectro/issues/858), [#859](https://github.com/fidals/shopelectro/issues/859) are still not solved.

**Comment by 0pdd** [comment]
@duker33 2 puzzles [#859](https://github.com/fidals/shopelectro/issues/859), [#863](https://github.com/fidals/shopelectro/issues/863) are still not solved; solved: [#858](https://github.com/fidals/shopelectro/issues/858).

**Comment by 0pdd** [comment]
@duker33 3 puzzles [#863](https://github.com/fidals/shopelectro/issues/863), [#864](https://github.com/fidals/shopelectro/issues/864), [#865](https://github.com/fidals/shopelectro/issues/865) are still not solved; solved: [#858](https://github.com/fidals/shopelectro/issues/858), [#859](https://github.com/fidals/shopelectro/issues/859).

**Comment by 0pdd** [comment]
@duker33 4 puzzles [#863](https://github.com/fidals/shopelectro/issues/863), [#865](https://github.com/fidals/shopelectro/issues/865), [#912](https://github.com/fidals/shopelectro/issues/912), [#913](https://github.com/fidals/shopelectro/issues/913) are still not solved; solved: [#858](https://github.com/fidals/shopelectro/issues/858), [#859](https://github.com/fidals/shopelectro/issues/859), [#864](https://github.com/fidals/shopelectro/issues/864).

**Comment by 0pdd** [comment]
@duker33 the puzzle [#863](https://github.com/fidals/shopelectro/issues/863) is still not solved; solved: [#858](https://github.com/fidals/shopelectro/issues/858), [#859](https://github.com/fidals/shopelectro/issues/859), [#864](https://github.com/fidals/shopelectro/issues/864), [#865](https://github.com/fidals/shopelectro/issues/865), [#912](https://github.com/fidals/shopelectro/issues/912), [#913](https://github.com/fidals/shopelectro/issues/913).

**Comment by 0pdd** [comment]
@duker33 all 7 puzzles are solved here: [#858](https://github.com/fidals/shopelectro/issues/858), [#859](https://github.com/fidals/shopelectro/issues/859), [#863](https://github.com/fidals/shopelectro/issues/863), [#864](https://github.com/fidals/shopelectro/issues/864), [#865](https://github.com/fidals/shopelectro/issues/865), [#912](https://github.com/fidals/shopelectro/issues/912), [#913](https://github.com/fidals/shopelectro/issues/913).
